src/app/modules/rup/components/elementos/rupers/seccionado/seccion.component.ts

```diff
@@ -259,7 +227,9 @@ export class SeccionComponent extends RUPComponent implements OnInit, OnDestroy
         }
         // Controlar si lo que llega como parámetro es un registro o es un concepto
         if (!registroOrigen.concepto) {
```

Comment:
esta condición queda vacia?

Reply:
Ahí me falto más limpieza. Porque es código muerto desde hace mucho o que se perdio funcionalidad en algún momento. 
Antiguamente se ve que podias drag and drop un concepto desde el buscador y arrojarlo dentro de otro concepto y ya te lo agregaba con la relación.

Puedo tratar de fixearlo o eliminar la funcionalidad entera.

Reply:
En definitiva nunca llega con registroOrigen vacío. Desde antes de este PR.